### PR TITLE
[PWN-8902] Local state

### PR DESCRIPTION
**Pull request by lisemyon**
## Link jira to issue
https://p2pvalidator.atlassian.net/browse/PWN-8902

## Description of the changes
-

### CODE REVIEW

File: Packages/KeyAppKit/Sources/BankTransfer/Models/UserCommonInfo.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
+public struct UserCommonInfo: Codable {
+    public let firstName: String?
+    public let lastName: String?
+    public let nationality: String?
```
Comment (sidorov-panda) [inline Packages/KeyAppKit/Sources/BankTransfer/Models/UserCommonInfo.swift:4]: can you add a comment, what type of data is here alpha2/3 or whatever, so we won't forget, maybe in future we will add a condition to not be able to set incorrect data

File: Packages/KeyAppKit/Sources/BankTransfer/Models/UserCommonInfo.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    public let placeOfBirth: String?
+    public let dateOfBirth: DateOfBirth?
+
+    public init(firstName: String?, lastName: String?, nationality: String?, placeOfBirth: String?, dateOfBirth: DateOfBirth?) {
```
Comment (sidorov-panda) [inline Packages/KeyAppKit/Sources/BankTransfer/Models/UserCommonInfo.swift:8]: is it necessary to add an init method here?

File: Packages/KeyAppKit/Sources/BankTransfer/Models/UserWallet.swift [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    public let walletId: String
+    public var accounts: UserAccounts
+}
+
+public struct UserAccounts: Codable {
+    public var eur: UserEURAccount?
+    public var usdc: UserUSDCAccount?
+}
+
+public struct UserEURAccount: Codable {
```
Comment (sidorov-panda) [inline Packages/KeyAppKit/Sources/BankTransfer/Models/UserWallet.swift:11]: do you consider renaming to EURUserAccount? or it's incorrect?

**Review by sidorov-panda — APPROVED** [review]

File: Packages/KeyAppKit/Sources/BankTransfer/Models/UserWallet.swift (hunk 3 above)
Comment (lisemyon) [reply Packages/KeyAppKit/Sources/BankTransfer/Models/UserWallet.swift:11]: i don't mind. i am not sure what is better

Comment (lisemyon) [reply Packages/KeyAppKit/Sources/BankTransfer/Models/UserWallet.swift:11]: i thought as we have UserAccounts, and User is like a namespace so it should be first

**Review by sidorov-panda — APPROVED** [review]

**Pull request merged**